### PR TITLE
Copy models from molo-ndoh-yep over.

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
tests passing but needs https://github.com/praekelt/molo/pull/33 to land.

**Comment by smn** [comment]
@miltontony let me know if you're happy with the general approach.

File: molo/profiles/views.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+from django.core.urlresolvers import reverse
+from django.shortcuts import render
+from django.contrib import messages
+from django.contrib.auth.models import User
+from django.utils.translation import ugettext_lazy as _
+
+
+@csrf_protect
+def register(request):
+    print 'register'
```
Comment (miltontony) [inline molo/profiles/views.py]: stray print

File: molo/profiles/views.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            return HttpResponseRedirect(form.cleaned_data.get('next', '/'))
+        print 'not valid'
+        return render(request, 'registration/register.html', {'form': form})
+    print 'get?'
+    form = RegistrationForm()
+    return render(request, 'registration/register.html', {'form': form})
+
+
+def register_success(request):
+    return HttpResponseRedirect(reverse('home_page'))
```
Comment (miltontony) [inline molo/profiles/views.py]: This has been replace by the `root_page.url`

**Comment by miltontony** [comment]
:ok_hand: on the general approach

**Comment by miltontony** [comment]
:+1:

**Pull request merged**